data-raw/watershed/yuba/yuba_river.rmd

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Yuba River"
 author: "[Sadie Gill](mailto:sgill@flowwest.com), [Mark Tompkins](mailto:mtompkins@flowwest.com)"
```

Comment:
Per the comment on mine, should we remove Sadie here?

Reply:
yes, i'll remove! i'll also put the suitability criteria in table format to be more aligned with your markdown